### PR TITLE
Use color map as background of transfer function control

### LINKED ISSUE #4: Transfer function canvas's Background Color
The transfer function canvas should have the current color map as background.

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- The canvas in `TransferFunctionControls.jsx` now uses the color map as it's background
- Colored the transfer function control's plot to match the sliders

The colors I matched are [here](https://github.com/react-component/slider/blob/master/assets/index.less).

I'm not fully sold on the color of the plot right now. Perhaps a deeper blue? Or maybe just go with something grayscale? I'd appreciate some feedback on the styling.

closes #4

### CODE REVIEW

**Comment by RobertGemmaJr** [comment]
closes #39

File: src/components/Controls/TransferFunctionControls.jsx [hunk 1]
```diff
@@ -249,9 +258,27 @@ function TransferFunctionControls({
   );
 }
 
-const OutlinedCanvas = styled.canvas`
-  width: 100%;
+const CanvasContainer = styled.div`
+  position: relative;
   outline: 1px solid;
+  /* cursor: ${(props) => props.cursor}; */
```
Comment (mcmcgrath13) [inline src/components/Controls/TransferFunctionControls.jsx:264]: delete? or un-comment?

File: src/components/Controls/TransferFunctionControls.jsx [hunk 2]
```diff
@@ -249,9 +258,27 @@ function TransferFunctionControls({
...
+`;
+
+const CanvasBackground = styled.div`
+  background-image: url(${(props) => props.src});
+  background-size: contain;
```
Comment (mcmcgrath13) [inline src/components/Controls/TransferFunctionControls.jsx:269]: 🌈

**Review by mcmcgrath13 — APPROVED** [review]
I think the colors matching the sliders looks good!

File: src/components/Controls/TransferFunctionControls.jsx (hunk 1 above)
Comment (RobertGemmaJr) [reply src/components/Controls/TransferFunctionControls.jsx:264]: deleted

**Pull request merged**